lstchain/tools/tests/test_create_cat_B_calibration_files.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+
+from ctapipe.core import run_tool
+from ctapipe_io_lst.constants import N_GAINS, N_PIXELS
+from ctapipe.io import read_table
+from lstchain.onsite import DEFAULT_CONFIG_CAT_B_CALIB 
+
+from lstchain.tests.test_lstchain import (
+    test_systematics_path,
+    test_calib_path
+)
+
+def test_create_catB_calibration_file(tmp_path,interleaved_r1_file):
+    '''Test the lstchain_create_cat_B_calibration_file tool'''
+    from lstchain.tools.lstchain_create_cat_B_calibration_file import CatBCalibrationHDF5Writer
+
+
+    input_path = interleaved_r1_file.parent
+    output_path = tmp_path / "calibration_cat_B_02006.h5"
+    stat_events = 5
+    print(f"config {DEFAULT_CONFIG_CAT_B_CALIB}")
+    
+    ret =  run_tool(
+        CatBCalibrationHDF5Writer(),
+        argv=[
+            f"--input_path={input_path}",
+            f"--output_file={output_path}",
+            f"--cat_A_calibration_file={test_calib_path}",
+            f"--LSTCalibrationCalculator.systematic_correction_path={test_systematics_path}",
+            f"--FlasherFlatFieldCalculator.sample_size={stat_events}",
+            f"--PedestalIntegrator.sample_size={stat_events}",
+            f"--config={DEFAULT_CONFIG_CAT_B_CALIB}",
+            "--overwrite",
+        ],
+        cwd=tmp_path,
+        )
+    
+    assert ret == 0, 'Running CalibrationHDF5Writer tool failed'
```

Comment:
ret == 0 and running failed? doesn't seem consistent

Reply:
The message in the right is the error message if the assert is false